deploy/configmap.yaml

```diff
@@ -21,4 +21,4 @@ data:
   Broadcom_bnxt_BCM57414_2x25G: "14e4 16d7 16dc"
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
   Qlogic_qede_QL45000_50G: "1077 1654 1664"
-  
+  Red_Hat_Virtio_network_device: "1af4" "1000" "0000"
```

Comment:
The vf and PF have the same deviceID it should be 1000 also for the VF

```
- Vfs:
      - deviceID: "1000"
        driver: vfio-pci
        pciAddress: "0000:00:04.0"
        vendor: 1af4
        vfID: 0
      deviceID: "1000"
      driver: vfio-pci
      netFilter: openstack/NetworkID:45da469e-6a3a-41f4-80e8-3c762dbd13b4
      numVfs: 1
      pciAddress: "0000:00:04.0"
      totalvfs: 1
      vendor: 1af4

```

Reply:
is it 0000 because of `vfID: 0`?

Reply:
the sriov-operator looks for the vendorID and  deviceID the vfID is just the number of the allocated VF by the kernel.
you can see that using the output

`ip link`